src/components/ppeItems/CustomModal.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { colors } from 'config'
+const closeIcon = require('assets/images/close-icon.png')
```

Comment:
This is a FontAwesome `times` icon so we can us that instead of the png.

Reply:
Yes we can use. I will update with FontAwesome icon

Reply:
Fixed, used Fontawesome icon. you can check now